PROME/Chapter3/src/Components/Movie.jsx

```diff
@@ -0,0 +1,42 @@
+import "../index.css";
+import { useState } from "react";
+function Movie(props) {

```

Comment:
```suggestion
function Movie({ name, overview, poster, date }) {
```
이렇게 구조분해할당을 사용하면 props를 계속 반복해서 사용해야 하는 번거로움을 없앨 수 있답니다